apps/examples/testcase/ta_tc/media/utc/utc_media_speechdetector.cpp

```diff
@@ -64,11 +64,35 @@ static void utc_media_SpeechDetector_initEndPointDetect_n(void)
 
 static void utc_media_SpeechDetector_deinitKeywordDetect_p(void)
 {
+	auto instance = media::voice::SpeechDetector::instance();
+	instance->initKeywordDetect(16000, 2);
```

Comment:
May be a #define would be more suitable for the values used
#define TEST_SAMPLE_RATE 16000
#define TEST_CHANNELS 2

Reply:
Thanks fro the review. I'll fix it